feature-configs/demo/gatekeeper/10_Constraints.yaml

```diff
@@ -0,0 +1,73 @@
+## This yaml *must* be loaded after 00_ConstraintTemplates.yaml, and after the gatekeeper operator
```

Comment:
This is not necessarily true.
The way the deployment works is that we iterate over all the yamls until everything works, so dependencies on CRDs are satisfied. That is the safe for the operators creating CRDs

Reply:
I agree.  It's true, and it's also not relevant.  The yaml must be loaded after the first one, but because you repeat/retry it will retry and load afterwards.  Agree that my comment can be updated to clarify, though.